### DISCUSSION TITLE
Is it possible to use DBeaver instead of PuTTY in this case?

### DISCUSSION THREAD

**Discussion by christesr75** [dbeaver/dbeaver#10980] [Q&A] [answered]
I use PuTTY(SSH) to connect to a Linux server from a Windows machine, and from PuTTY i connect to the databases with the commands "isql" for Firebird,"sqlplus" for Oracle and "fisql" for SQL. The PuTTY screen is ok for a small query but awful for most of the more serious stuff. Is it possible to connect to those databases directly from Windows using DBeaver?

I tried to connect with the DBeaver using +/- the same info i use on PuTTY but i only managed to make the SSH tunnel work.

**Comment by p91paul** [comment, marked as answer]
Make sure you are using ip addresses and not hostnames for your db, name resolution doesn't normally work through an ssh tunnel.

**Reply by Unterbix** [reply]
Thanks! Change the localhost to 127.0.0.1 helped me.
